path_2023.py

```diff
@@ -20,11 +20,11 @@
 CAMERA = 0
 ALTITUDE = 40
 CONNECTION_ADDRESS = "tcp:localhost:14550"
-DELAY = 0.1 # seconds
+DELAY = 0.1   # seconds
 
 
 def run() -> int:
-    drone = dronekit.connect(CONNECTION_ADDRESS, wait_ready = True)
+    drone = dronekit.connect(CONNECTION_ADDRESS, wait_ready = True, timeout = 60)
```

Comment:
Set `wait_ready` to false and add a comment (you can copy the comment in the common repository's flight controller).